src/core/Global.ttslua

```diff
@@ -42,7 +42,7 @@ local navigationOverlayApi = require("core/NavigationOverlayApi")
 
 -- online functionality related variables
 local MOD_VERSION = "3.1.0"
-local SOURCE_REPO = 'https://raw.githubusercontent.com/chr1z93/loadable-objects/main'
+local SOURCE_REPO = 'https://raw.githubusercontent.com/argonui/SCED/main'
```

Comment:
I don't love this pointing to the top level of the repo.  As mentioned in the other PR, let's make this look to content

Reply:
Updated